storybook/stories/dash/Users/EditUser.stories.tsx

```diff
@@ -47,11 +47,92 @@ export const Default: Story = props({
   addNewBlockedApp: () => {},
   removeBlockedApp: () => {},
   newBlockedAppIdentifier: ``,
+  blockedApps: [
+    { id: `1`, identifier: `Notepad++` },
+    {
+      id: `2`,
+      identifier: `Xcode`,
+      schedule: {
+        mode: `active`,
+        days: {
+          sunday: true,
+          monday: true,
+          tuesday: true,
+          wednesday: true,
+          thursday: true,
+          friday: true,
+          saturday: true,
+        },
+        window: {
+          start: {
+            hour: 6,
+            minute: 0,
+          },
+          end: {
+            hour: 17,
+            minute: 30,
+          },
+        },
+      },
+    },
+    {
+      id: `3`,
+      identifier: `Really really long app name`,
+      schedule: {
```

Comment:
same here

Reply:
I wanted this one to have a long schedule description and app name, so I think I'll leave it as is if that's ok